src/wallet/rpcwallet.cpp

```diff
@@ -2804,9 +2804,9 @@ UniValue bumpfee(const JSONRPCRequest& request)
             nNewFeeRate = CWallet::fallbackFee;
         }
 
-        // new fee rate must be at least old rate + minimum relay rate
-        if (nNewFeeRate.GetFeePerK() < nOldFeeRate.GetFeePerK() + ::minRelayTxFee.GetFeePerK()) {
```

Comment:
Consider keeping the :: prefix for global variables. As someone new to the code who doesn't have all the global variables memorized, I think the prefix is very helpful for readability, especially in a large function like bumpfee with similar sounding local variables.